scs-app/src/app/fulfill_major/fulfill_major.component.html

```diff
@@ -0,0 +1,48 @@
+<div id="fulfill-major-tool">
+
+<div id="title">Course Selection Tool</div><br>
+<div id="blurb">
+  <p>Get suggestions for classes that will complete your major requirements!
+    Add courses below to see which categories you still need to fulfill.
+    You can also enter the grades you've received to calculate your major GPA.</p>
+</div><br><br>
+
+<div class="flex-container">
```

Comment:
bootstrap has it's flex https://getbootstrap.com/docs/4.0/utilities/flex/ should we use this instead?